docs/sources/dashboards/use-dashboards/index.md

```diff
@@ -190,6 +190,20 @@ This is equivalent to the **Today so far** time range preset, but it starts at 8
 
 Using a semi-relative time range, as time progresses, your dashboard will automatically and progressively zoom out to show more history and fewer details. At the same rate, as high data resolution decreases, historical trends over the entire time period will become more clear.
 
+#### Copy and paste time range
+
+You can copy and paste time range from a dashboard to Explore and vice versa. You can also copy and paste time range from one dashboard to another.
+
+##### Copy time range (t+c)
+
+<img class="no-shadow" src="/media/docs/grafana/dashboards/screenshot-copy-paste-time-range.png" max-width="900">
+
+Click the **Copy time range to clipboard** icon to copy the current time range to the clipboard. You can then paste the time range into Explore.
+
+##### Paste time range (t+v)
+
+Click the **Paste time range** icon to paste the time range from the clipboard into the current time range. Currently
```

Comment:
Most people are familiar with the copy/paste function on a keyboard, so we don't need to be so detailed in this explanation or use as many headings.
```suggestion
Click the **Copy time range to clipboard** icon to copy the current time range to the clipboard. Then paste the time range into Explore or another dashboard.

<img class="no-shadow" src="/media/docs/grafana/dashboards/screenshot-copy-paste-time-range.png" max-width="900">

You can also use the keyboard shortcuts `t+c` and `t+v` to copy and paste respectively.
```